### DISCUSSION TITLE
Is postgres a database feasible for use with Nima ?

### DISCUSSION THREAD

**Discussion by geraldodev** [helidon-io/helidon#8590] [Q&A]
Hi, I'm postgres database user and want to know if is it feasible to use postgres with Nima. I don't think so. Postgres jdbc driver seems to have solved all issues with syncronized, but what about the connection pools. I'm assuming that I need one to limit the number of connections made to the database.  As far I can see it's an open issue to hikaricp.
Can you educate me on this matter ?

**Comment by klustria** [comment]
Unfortunately, we'll have to wait for HikariCP to resolve this issue.
